src/main/java/net/minecraftforge/fml/client/SplashProgress.java

```diff
@@ -357,7 +357,11 @@ public void run()
                         clearGL();
                         setGL();
                     }
-                    Display.sync(100);
+                    try
+                    {
+                        Thread.sleep(10);
```

Comment:
30 ms would keep it close to 33fps, 10 ms roughly matches the 100fps used before.